tbx/project_styleguide/templates/patterns/styleguide_base.html

```diff
@@ -12,9 +12,10 @@
             <label for="theme-chooser" class="text-heading">Choose a theme:</label>
             <select name="theme-chooser" id="theme-chooser" class="p-2 text-black select" onchange="changeThemeAndMode()">
                 <option value="theme-coral">Coral</option>
+                <option value="theme-nebuline">Nebuline</option>
                 <option value="theme-lagoon">Lagoon</option>
-                <option value="theme-banana">Banana</option>
-                <option value="theme-earth">Earth</option>
+                <option value="theme-green">Green</option>
+                <option value="">None</option>
```

Comment:
Q: Is this being added because there are some pages that don't have any theme specific styles?

Reply:
yes, we now have this option - it is used on the home page (like, the avatar decor is white instead of using themed colors)